packages/diracx-web-components/tsup.config.ts

```diff
@@ -0,0 +1,61 @@
+import * as fs from "node:fs/promises";
+import * as path from "path";
+import { defineConfig } from "tsup";
+
+export default defineConfig([
+  {
+    entry: [
+      "components/!(index|*.stories).ts?(x)",
+      "components/*/!(index|*.stories).ts?(x)",
+      "hooks/!(index|*.stories).ts?(x)",
+      "contexts/!(index|*.stories).ts?(x)",
+      "types/!(index|*.stories).ts?(x)",
+    ],
+    format: ["esm", "cjs"],
+    experimentalDts: true, // Seems to work fine, lower memory usage and faster than dts
+    target: "es5",
```

Comment:
There's about a 0.4% difference in size in my testing (4kB out of 1.1MB) so it seems pretty negligible.

I don't know about performance though, so it could be a good idea to keep it anyway.
Thoughts?

Reply:
Indeed, it is negligible. Let's try with `es6` anyway, we will adjust later if we really need `es5` but I really doubt it will be the case.